### DISCUSSION TITLE
English title عنوان فارسی

### DISCUSSION THREAD

**Discussion by hiddify-com** [hiddify/Hiddify-Manager#105] [Q&A سوالات رایج] [answered]
چگونه فلان کار را انجام بدیم
How to do some action

**Comment by hiddify-com** [comment]
اینجوری

**Comment by hiddify-com** [comment]
کامنت اضافه

**Reply by just1random** [reply]
@hiddify If it is ok with you, I will volunteer to help you answer some of the non-technical issues I know about. Maintaining the old issues and closing them with proper responses, updating some of the docs in both languages, answering the questions in the discussion, and selecting the correct answer provided by others. I can help you organize this repository in the parts I know about.

**Comment by hiddify-com** [comment, marked as answer]
In this way

به این شکل
